google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/CloseStream.java

```diff
@@ -50,8 +50,8 @@ private CloseStream(Status status, List<StreamContinuationToken> continuationTok
   }
 
   @InternalApi("Used in Changestream beam pipeline.")
-  public Status getStatus() {
-    return this.status;
+  public com.google.cloud.bigtable.common.Status getStatus() {
```

Comment:
why long import?

Reply:
Because I already import `import com.google.rpc.Status;`. CloseStream holds the proto type status (i.e. `com.google.rpc.Status`), and `getStatus()` return the wrapper class(i.e. `com.google.cloud.bigtable.common.Status`)